### PR TITLE
Document how to implement mixin commands

### LINKED ISSUE #216: Document how to make a mixin schema
See https://github.com/deislabs/porter-helm/pull/22#issuecomment-471542715

There are some important rules and tips on making a json schema for a mixin that we should provide for anyone making a mixin. I wrote them out as I learned them in the PR above while redoing the helm mixin. As soon as that's done, that should be turned into guidance on a new page:

Developing Mixins > Commands

This page will eventually cover other commands as well like `build`, `run`, etc.

### PR DESCRIPTION
**Pull request by carolynvs**
Got excited and documented all the commands.

Closes #216

Preview available at https://deploy-preview-221--porter.netlify.com/mixin-commands/

### CODE REVIEW

**Comment by carolynvs** [comment]
@jeremyrickard Let me know how this matched up to what you had to implement for the kubernetes mixin.

**Comment by jeremyrickard** [comment]
That checks out pretty well. Couple of small comments inline

File: docs/content/mixin-commands.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+**stdin**
+```yaml
+status:
+- helm:
+    description: "Status of MySQL"
+    releases:
+    - porter-ci-mysql
+
+# version
```
Comment (jeremyrickard) [inline docs/content/mixin-commands.md]: missing a close to the previous YAML block, so this doesn't get hyperlinked.

File: docs/content/mixin-commands.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+**/cnab/app/porter/outputs/mixin-output-abc123**
+```
+password=topsecret
+host=127.0.0.1
+```
+
+# status
+
+The status command (required) is called from inside the invocation image during
```
Comment (jeremyrickard) [inline docs/content/mixin-commands.md]: status is an optional command isn't it?

**Comment by carolynvs** [comment]
Good catch, fixed!

**Comment by carolynvs** [comment]
CI is stuck but the doc changes are in. Merging with my mega root. 💪

**Pull request merged**